public/src/actions/script.jsx

```diff
@@ -0,0 +1,46 @@
+import axios from 'axios';
+import { SET_SCRIPT_INFO, SET_SCRIPT_QUESTIONS } from '../reducers/script';
+
+export function setScriptInfo(script) {
+  return {
+    type: SET_SCRIPT_INFO,
+    payload: script
+  };
+}
+
+export function setScriptQuestions(questions) {
+  return {
+    type: SET_SCRIPT_QUESTIONS,
+    payload: questions
+  };
+}
+
+export function fetchScript(id) {
+  return dispatch => axios.get(`/scripts/${id}`)
+    .then((res) => {
+      const scriptData = res.data;
```

Comment:
Could use deconstruction for this:
const { data: scriptData } = res;
But totally not necessary.